synapse/cores/common.py

```diff
@@ -308,6 +457,11 @@ def isRuntProp(self, prop):
     def _cortex_condefs():
         confdefs = (
 
+            ('dir', {'type': 'str',
```

Comment:
Do we want to start namespacing new Cortex config options?  ``cortex:dir`` and ``cortex:fifo:defs`` ?